source/usage-examples/watch.txt

```diff
@@ -0,0 +1,139 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
```

Comment:
Nitpick: feel free to disagree, but when using the imperative form in these headings, it reads to me more as a procedural tutorial (which you might find in the Fundamentals section) than a usage example. Since the other Usage Examples use an approach just to list information, I wonder if it might be better to adhere to that format.